### DISCUSSION TITLE
No github READMEs sample profiles is visible on Recode Hive

### DISCUSSION THREAD

**Discussion by sitambhra** [recodehive/awesome-github-profiles#557] [Q&A]
![image](https://github.com/user-attachments/assets/16ae5dbb-df0f-4b61-ba9c-f6834384ff62)
Hi there!
I opened the link given in the about section of 'awesome-github-profiles' repository. However, no profile is showing up. What to do?

**Comment by sanjay-kv** [comment]
there is some issue happening with latest merge. team is working to fix it.
